recipes/baybe/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "baybe" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Upstream published wheels only. You should request them to publish the source distribution (.tar.gz)  or see if they have releases on GitHub or something similar.

Reply:
Hi @ocefpaf, thanks for your help. For clarity: we are the creators of the package and released it's first public version just yesterday. So in principle, we could publish the source distribution as well, if it helps!

However, we also have a first release in Github, and I tried using that with `grayskull`. I've just updated the recipe to what `grayskull` produced, but this brings me back to my original question how to properly account for our optional dependencies. In fact, `grayskull` immediately run into a problem since we are referencing one of the dependency groups in the main deps:
<img width="398" alt="image" src="https://github.com/conda-forge/staged-recipes/assets/23265127/9e29eca9-d0da-4f0c-86ad-7d16b41171ce">

Also, none of the other groups are included...